Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/TextToolbar.Events.cs

```diff
@@ -45,7 +46,8 @@ private static void OnEditorChanged(DependencyObject obj, DependencyPropertyChan
                 if (newEditor != null)
                 {
                     newEditor.AddHandler(KeyDownEvent, bar.KeyEventHandler, handledEventsToo: true);
-                    bar.CreateFormatter();
+                    bar.Formatter.SetModel(bar);
+                    bar.DefaultButtons = bar.Formatter.DefaultButtons;
```

Comment:
Null checks `?.` for Formatter here?

Also, above we can do pattern matching for `if (obj is TextToolbar bar)`

Reply:
I've applied pattern matching for this whole control, and in this particular case, I've change that if, in case the Formatter is null.